app/docs/dev/policies/targetref.md

```diff
@@ -0,0 +1,333 @@
+---
+title: Understanding TargetRef policies
+---
+
+## What is a policy?
+
+A policy is anything that can modify the configuration of a proxy.
```

Comment:
This definition is a little vague for me. If I were unfamiliar with Kuma and it's policies, I wouldn't understand this definition. [Here](https://kuma.io/docs/1.8.x/explore/dpp/#dynamic-configuration-of-the-data-plane-proxy) docs say that policies are used to generate DP configuration. So maybe something like this: 
`A policy is a set of guidelines that will be used to build proxy configuration. Kuma combines policies and Dataplane entities to build configuration for proxy`